sdks/python/build.gradle

```diff
@@ -283,13 +283,14 @@ def portableWordCountTask(name, streaming, envdir) {
               "--input=/etc/profile",
               "--output=/tmp/py-wordcount-direct",
               "--runner=PortableRunner",
-              "--experiments=worker_threads=100",
       ]
       if (streaming)
         options += ["--streaming"]
       else
       // workaround for local file output in docker container
         options += ["--environment_cache_millis=10000"]
+      // [BEAM-5167] Workaround for scheduling issue between SDKHarness and Flink
+        options += ["--parallelism=1"]
```

Comment:
Good catch. That's why it's better to always use brackets for if blocks.

Reply:
Ahh, Totally missed it. BTW python works with indentation so it might have worked with Python :)

Should also apply this change to streaming as we don't want parallelism in streaming as well.

Reply:
Wouldn't make this more restrictive than necessary. So only batch for now.